docs/administration/configuration/configuration.md

```diff
@@ -36,6 +36,10 @@ These files contain additional settings and point to the general (not environmen
 
     Read more about [how configuration is handled in Symfony]([[= symfony_doc =]]/best_practices/configuration.html).
 
+!!! caution
+
+    Avoid using special characters in your configuration files. Make sure your IDE displays them. Be careful when copy-pasting from a word processing software or a PDF (it might content hidden character like [soft hyphen](https://en.wikipedia.org/wiki/Soft_hyphen).
```

Comment:
@lionelakpagni You're right about ambiguity and `#` or even `-` which can be considered as special characters. What do you think about this precision?:
```suggestion
    Avoid using Unicode characters from the ["Other" (`C`) categories (such as control or format characters)](https://en.wikipedia.org/wiki/Unicode#General_Category_property) in your configuration files. Make sure your IDE displays them. Be careful when copy-pasting from a word processing software or a PDF (it might content hidden character like [soft hyphen](https://en.wikipedia.org/wiki/Soft_hyphen).
```

Reply:
Maybe we should also keep "special" here? as in "avoid using special or Unicode characters" ?
#¤%&( are for instance special characters, but not unicode characters. And they should be avoided

Reply:
Using special as general will mean avoiding also` _`  ,` -`  
But its true that some special should be avoied,  at least we may mention to preferably Quote them **''**.

[Symfony doc](https://symfony.com/doc/current/reference/formats/yaml.html#strings)
> Quoted styles are useful when a string starts or end with one or more relevant spaces, because unquoted strings are trimmed on both end when parsing their contents. Quotes are required when the string contains special or reserved characters.

Reply:
@vidarl
I can keep "special" this way:
```suggestion
    Avoid using special characters in your configuration files, avoid especially Unicode characters from the ["Other" (`C`) categories (such as control or format characters)](https://en.wikipedia.org/wiki/Unicode#General_Category_property). Make sure your IDE displays them. Be careful when copy-pasting from a word processing software or a PDF (it might content hidden character like [soft hyphen](https://en.wikipedia.org/wiki/Soft_hyphen).
```
(insistence about Unicode "Other" comes from https://github.com/thephpleague/flysystem/blob/3.x/src/WhitespacePathNormalizer.php#L19 `\p{C}`)